osu.Game/Screens/Edit/Components/Timelines/Summary/Parts/PreviewTimePart.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Game.Graphics;
+using osu.Game.Screens.Edit.Components.Timelines.Summary.Visualisations;
+
+namespace osu.Game.Screens.Edit.Components.Timelines.Summary.Parts
+{
+    public partial class PreviewTimePart : TimelinePart
+    {
+        protected override void LoadBeatmap(EditorBeatmap beatmap)
+        {
+            base.LoadBeatmap(beatmap);
+            Add(new PreviewTimeVisualisation(beatmap.PreviewTime));
```

Comment:
also just noticed that this is incorrect, you cannot pass bindables like this to component - make a local bindable in `PreviewTimeVisualisation` and bind to `beatmap.PreviewTime` instead